packages/@groupby/elements-search-driver-plugin/src/search-driver-plugin.ts

```diff
@@ -92,16 +96,23 @@ export default class SearchDriverPlugin<P = Record> implements Plugin {
    * and emits the result through an event. The result is emitted in a
    * [[SEARCH_RESPONSE]] event. If the search fails for any
    * reason, a [[SEARCH_ERROR]] is dispatched with the error.
+   * Emits a beacon event if search is successful.
    *
    * @param event the event whose payload is the search term.
    */
   fetchSearchData(event: CustomEvent<SearchRequestPayload>): void {
-    const { query, group, config } = event.detail;
+    const {
+      query,
+      group,
+      config,
+      origin,
+    } = event.detail;
     this.sendSearchApiRequest({ query, ...config })
-      .then((results) => {
-        const payload: SearchResponsePayload<P> = { ...results, group };
+      .then((response) => {
+        const payload: SearchResponsePayload<P> = { ...response, group };
```

Comment:
For consistency, the SAYT driver still uses `results` in its SAYT and Search calls. Should update those to the same.

Reply:
I have updated the matching function in the Sayt Driver, but not the fetchAutocomplete function for reasons of simplicity.